ui/src/features/common/promotion-directive-step-status/utils.ts

```diff
@@ -30,7 +33,7 @@ export const getPromotionDirectiveStepStatus = (
 
   if (
     promotionStatus?.phase !== PromotionStatusPhase.SUCCEEDED &&
-    stepNumber > Number(promotionStatus?.currentStep)
+    stepNumber >= Number(promotionStatus?.currentStep)
```

Comment:
@hiddeco can you confirm what is index of promotion step after it transition to Aborted state?

Reply:
It will freeze.

Reply:
Thanks for confirming. The change in this line is correct!